example-crates/wallet_electrum/src/main.rs

```diff
@@ -26,7 +26,9 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
         Network::Testnet,
     )?;
 
-    let address = wallet.get_address(bdk::wallet::AddressIndex::New);
+    let address = wallet
+        .try_get_address(bdk::wallet::AddressIndex::New)
+        .expect("new_address");
```

Comment:
You should not panic when this fails. It indicates that persistence failed. A good way to do this might be to use `anyhow::Context::context` and then `?` the error. It doesn't look like the examples depend on `anyhow` but they should and the main function should return `anyohw::Result`.

Reply:
I added `anyhow` and removed the `.expect()` but didn't add the `.context()` because I don't think it's needed for this simple example.